Index: src/Normalizer/ContentEntityNormalizer.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/Normalizer/ContentEntityNormalizer.php	(date 1542837137000)
+++ src/Normalizer/ContentEntityNormalizer.php	(date 1542837137000)
@@ -153,7 +153,7 @@
         $key = $field_definition->getFieldStorageDefinition()
                                 ->getMainPropertyName();
         foreach ($data[$field_name] as &$item) {
-          if (!empty($item[$key_entityid]) && !empty($item['target_uuid'])) {
+          if (!empty($item['target_uuid'])) {
             $reference = $this->entityManager->loadEntityByUuid($item['target_type'], $item['target_uuid']);
             if ($reference) {
               $item[$key] = $reference->id();
